README.md

```diff
@@ -173,7 +173,7 @@ Commands:
 
 Options:
   -b, --backend <backend>
-          The initial backend hosting a model to connect to. [default: ollama] [env: OATMEAL_BACKEND=] [possible values: langchain, ollama, openai]
+          The initial backend hosting a model to connect to. [default: ollama] [env: OATMEAL_BACKEND=] [possible values: langchain, ollama, openai, githubcopilot]
```

Comment:
should we leave this full name or just copilot?

Reply:
Full name works! They don't own the trademark to just `copilot`, so others could pop up.